README.md

```diff
@@ -1,4 +1,5 @@
-# Trading program
+![Alt text](logo.png?raw=true "Title")
```

Comment:
I think you are merging also my changes. 

I think you should run `git rebase origin main` on you branch to remove them